string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-unused-vars */
+
+function capitalizeWord(word) {
+  if (word.length === 10) {
+    return 'JavaScript';
+  }

```

Comment:
I don't recommend checking the length here. This will fail for words with 10 letters that aren't JavaScript.

Reply:
Alrighty. Yeah, it was a bad solution, but I just stuck on trying to figure out a more versatile solution. I'll work on it.